platform/android/app/src/main/kotlin/coop/polypoly/polypod/Authentication.kt

```diff
@@ -20,6 +22,12 @@ class Authentication {
                 !Preferences.isFirstRun(context)
         }
 
+        fun should_authenticate(context: Context): Boolean {
```

Comment:
This isn't Rust yet :D Should be `shouldAuthenticate`,  right?

Reply:
... just switched from Rust 😄